src/test/java/api/requests/StaffSlipsTests.java

```diff
@@ -89,51 +99,89 @@ void responseContainsNoPickSlipsForWrongServicePointId() {
       .by(usersFixture.james()));
 
     UUID differentServicePointId = servicePointsFixture.cd2().getId();
-    Response response = ResourceClient.forPickSlips()
-      .getById(differentServicePointId);
+    Response response = slipsType.get(differentServicePointId);
 
     assertThat(response.getStatusCode(), is(HTTP_OK));
-    assertResponseHasItems(response, 0);
+    assertResponseHasItems(response, 0, slipsType);
   }
 
-  @Test
-  void responseContainsNoPickSlipsWhenThereAreNoPagedItems() {
+  @ParameterizedTest
+  @EnumSource(value = SlipsType.class)
+  void responseContainsNoSlipsWhenThereAreNoItems(SlipsType slipsType) {
     UUID servicePointId = servicePointsFixture.cd1().getId();
-    Response response = ResourceClient.forPickSlips().getById(servicePointId);
+    Response response = slipsType.get(servicePointId);
 
     assertThat(response.getStatusCode(), is(HTTP_OK));
-    assertResponseHasItems(response, 0);
+    assertResponseHasItems(response, 0, slipsType);
   }
 
-  @Test
-  void responseContainsNoPickSlipsWhenItemHasOpenPageRequestWithWrongStatus() {
+  @ParameterizedTest
+  @EnumSource(value = SlipsType.class)
+  void responseContainsNoPickSlipsWhenItemHasOpenRequestWithWrongStatus(SlipsType slipsType) {
     UUID servicePointId = servicePointsFixture.cd1().getId();
     ItemResource item = itemsFixture.basedUponSmallAngryPlanet();
 
+    if (slipsType == SlipsType.SEARCH_SLIPS) {
+      checkOutFixture.checkOutByBarcode(item);
+    }
+
     requestsFixture.place(new RequestBuilder()
-      .page()
+      .withRequestType(slipsType.getRequestType().getValue())
+      .withStatus(RequestStatus.OPEN_AWAITING_PICKUP.getValue())
+      .withPickupServicePointId(servicePointId)
+      .forItem(item)
+      .by(usersFixture.james()));
+
+    Response response = slipsType.get(servicePointId);
+
+    assertThat(response.getStatusCode(), is(HTTP_OK));
+    assertResponseHasItems(response, 0, slipsType);
+  }
+
+  @ParameterizedTest
+  @MethodSource(value = "getAllowedStatusesForHoldRequest")
```

Comment:
what if we just explicitly exclude NONE value from @EnumSource?

Reply:
And which enum would we use? `ItemStatus` contains many statuses which are not allowed for a Hold request.